Versions after v3.3.37 doesn't work on Windows 7

With version after v3.3.37, NBGV fails when doing a simple dotnet build in new repo.

Error is as follows:
>error MSB4018: The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
error MSB4018: System.EntryPointNotFoundException: Unable to find an entry point named 'CreateFile' in DLL 'api-ms-win-core-file-l1-2-0.dll'.
error MSB4018:    at PInvoke.Kernel32.CreateFile(Char* filename, ACCESS_MASK access, FileShare share, SECURITY_ATTRIBUTES* securityAttributes, CreationDisposition creationDisposition, CreateFileFlags flagsAndAttributes, SafeObjectHandle templateFile)
error MSB4018:    at PInvoke.Kernel32.CreateFile(String filename, ACCESS_MASK access, FileShare share, SECURITY_ATTRIBUTES* securityAttributes, CreationDisposition creationDisposition, CreateFileFlags flagsAndAttributes, SafeObjectHandle templateFile)
error MSB4018:    at PInvoke.Kernel32.CreateFile(String filename, ACCESS_MASK access, FileShare share, Nullable`1 securityAttributes, CreationDisposition creationDisposition, CreateFileFlags flagsAndAttributes, SafeObjectHandle templateFile)
error MSB4018:    at Nerdbank.GitVersioning.ManagedGit.FileHelpers.TryOpen(String path, FileStream& stream)
error MSB4018:    at Nerdbank.GitVersioning.ManagedGit.GitRepository..ctor(String workingDirectory, String gitDirectory, String commonDirectory, String objectDirectory)
error MSB4018:    at Nerdbank.GitVersioning.ManagedGit.GitRepository.Create(String workingDirectory)
error MSB4018:    at Nerdbank.GitVersioning.Managed.ManagedGitContext..ctor(String workingDirectory, String dotGitPath, String committish)
error MSB4018:    at Nerdbank.GitVersioning.GitContext.Create(String path, String committish, Boolean writable)
error MSB4018:    at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()

For full error, see last step.

Running command ```ver``` on my machine
> Microsoft Windows [Version 6.1.7601]

Steps to reproduce
1. dotnet --version
> 3.1.410
2. dotnet new console
3. git init
4. git add -A && git commit -m "init"
5. dotnet tool install -g nbgv
5. nbgv install
6. git add -A && git commit -m "nbgv"
7. dotnet build

> Microsoft (R) Build Engine version 16.7.2+b60ddb6f4 for .NET
Copyright (C) Microsoft Corporation. All rights reserved.
>
>  Determining projects to restore...
>  All projects are up-to-date for restore.
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018: The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018: System.EntryPointNotFoundException: Unable to find an entry point named 'CreateFile' in DLL 'api-ms-win-core-file-l1-2-0.dll'.
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at PInvoke.Kernel32.CreateFile(Char* filename, ACCESS_MASK access, FileShare share, SECURITY_ATTRIBUTES* securityAttributes, CreationDisposition creationDisposition, CreateFileFlags flagsAndAttributes, SafeObjectHandle templateFile)
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at PInvoke.Kernel32.CreateFile(String filename, ACCESS_MASK access, FileShare share, SECURITY_ATTRIBUTES* securityAttributes, CreationDisposition creationDisposition, CreateFileFlags flagsAndAttributes, SafeObjectHandle templateFile)
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at PInvoke.Kernel32.CreateFile(String filename, ACCESS_MASK access, FileShare share, Nullable`1 securityAttributes, CreationDisposition creationDisposition, CreateFileFlags flagsAndAttributes, SafeObjectHandle templateFile)
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at Nerdbank.GitVersioning.ManagedGit.FileHelpers.TryOpen(String path, FileStream& stream)
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at Nerdbank.GitVersioning.ManagedGit.GitRepository..ctor(String workingDirectory, String gitDirectory, String commonDirectory, String objectDirectory)
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at Nerdbank.GitVersioning.ManagedGit.GitRepository.Create(String workingDirectory)
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at Nerdbank.GitVersioning.Managed.ManagedGitContext..ctor(String workingDirectory, String dotGitPath, String committish)
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at Nerdbank.GitVersioning.GitContext.Create(String path, String committish, Boolean writable)
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
>
>Build FAILED.
>
>C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018: The "Nerdbank.GitVersioning.Tasks.GetBuildVersion" task failed unexpectedly.
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018: System.EntryPointNotFoundException: Unable to find an entry point named 'CreateFile' in DLL 'api-ms-win-core-file-l1-2-0.dll'.
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at PInvoke.Kernel32.CreateFile(Char* filename, ACCESS_MASK access, FileShare share, SECURITY_ATTRIBUTES* securityAttributes, CreationDisposition creationDisposition, CreateFileFlags flagsAndAttributes, SafeObjectHandle templateFile)
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at PInvoke.Kernel32.CreateFile(String filename, ACCESS_MASK access, FileShare share, SECURITY_ATTRIBUTES* securityAttributes, CreationDisposition creationDisposition, CreateFileFlags flagsAndAttributes, SafeObjectHandle templateFile)
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at PInvoke.Kernel32.CreateFile(String filename, ACCESS_MASK access, FileShare share, Nullable`1 securityAttributes, CreationDisposition creationDisposition, CreateFileFlags flagsAndAttributes, SafeObjectHandle templateFile)
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at Nerdbank.GitVersioning.ManagedGit.FileHelpers.TryOpen(String path, FileStream& stream)
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at Nerdbank.GitVersioning.ManagedGit.GitRepository..ctor(String workingDirectory, String gitDirectory, String commonDirectory, String objectDirectory)
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at Nerdbank.GitVersioning.ManagedGit.GitRepository.Create(String workingDirectory)
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at Nerdbank.GitVersioning.Managed.ManagedGitContext..ctor(String workingDirectory, String dotGitPath, String committish)
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at Nerdbank.GitVersioning.GitContext.Create(String path, String committish, Boolean writable)
C:\Users\user\\.nuget\packages\nerdbank.gitversioning\3.4.205\build\Nerdbank.GitVersioning.Inner.targets(17,5): error MSB4018:    at Nerdbank.GitVersioning.Tasks.GetBuildVersion.ExecuteInner()
>    0 Warning(s)
    1 Error(s)
>
>Time Elapsed 00:00:01.14
>

--- a/3rdPartyNotices.txt
+++ b/3rdPartyNotices.txt
@@ -127,31 +127,6 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
-dotnet/PInvoke (https://github.com/dotnet/pinvoke)
-==============
-
-MIT License
-
-Copyright (c) .NET Foundation and Contributors
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
-
 Cake Contrib (https://github.com/cake-contrib/graphics)
 ============
 

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -58,13 +58,13 @@ stages:
       displayName: Configure git commit author for testing
 
     - task: UseDotNet@2
-      displayName: Install .NET Core SDK 5.0.202
+      displayName: Install .NET Core 5.0.202 SDK
       inputs:
         packageType: sdk
         version: 5.0.202
 
     - task: UseDotNet@2
-      displayName: Install .NET Core 3.1
+      displayName: Install .NET Core 3.1 runtime
       inputs:
         packageType: runtime
         version: 3.1.x
@@ -296,17 +296,17 @@ stages:
       vmImage: $(imageName)
     steps:
     - task: UseDotNet@2
-      displayName: Install .NET Core SDK 2.1.811
+      displayName: Install .NET Core 2.1 runtime
       inputs:
-        packageType: sdk
-        version: 2.1.811
+        packageType: runtime
+        version: 2.1.x
     - task: UseDotNet@2
-      displayName: Install .NET Core SDK 3.1.100
+      displayName: Install .NET Core 3.1 runtime
       inputs:
-        packageType: sdk
-        version: 3.1.100
+        packageType: runtime
+        version: 3.1.x
     - task: UseDotNet@2
-      displayName: Install .NET Core SDK 5.0.202
+      displayName: Install .NET Core 5.0.202 SDK
       inputs:
         packageType: sdk
         version: 5.0.202

--- a/src/NerdBank.GitVersioning/ManagedGit/FileHelpers.cs
+++ b/src/NerdBank.GitVersioning/ManagedGit/FileHelpers.cs
@@ -5,7 +5,9 @@ using System.Diagnostics.CodeAnalysis;
 using System.IO;
 using System.Runtime.InteropServices;
 using Microsoft.Win32.SafeHandles;
-using Microsoft.Windows.Sdk;
+using Windows.Win32;
+using Windows.Win32.Storage.FileSystem;
+using Windows.Win32.System.SystemServices;
 
 namespace Nerdbank.GitVersioning.ManagedGit
 {
@@ -23,7 +25,7 @@ namespace Nerdbank.GitVersioning.ManagedGit
         {
             if (IsWindows)
             {
-                var handle = PInvoke.CreateFile(path, FILE_ACCESS_FLAGS.FILE_GENERIC_READ, FILE_SHARE_FLAGS.FILE_SHARE_READ, lpSecurityAttributes: null, FILE_CREATE_FLAGS.OPEN_EXISTING, FILE_FLAGS_AND_ATTRIBUTES.FILE_ATTRIBUTE_NORMAL, null);
+                var handle = PInvoke.CreateFile(path, FILE_ACCESS_FLAGS.FILE_GENERIC_READ, FILE_SHARE_MODE.FILE_SHARE_READ, lpSecurityAttributes: null, FILE_CREATION_DISPOSITION.OPEN_EXISTING, FILE_FLAGS_AND_ATTRIBUTES.FILE_ATTRIBUTE_NORMAL, null);
 
                 if (!handle.IsInvalid)
                 {
@@ -64,7 +66,7 @@ namespace Nerdbank.GitVersioning.ManagedGit
                 HANDLE handle;
                 fixed (char* pPath = &path[0])
                 {
-                    handle = PInvoke.CreateFile(pPath, FILE_ACCESS_FLAGS.FILE_GENERIC_READ, FILE_SHARE_FLAGS.FILE_SHARE_READ, null, FILE_CREATE_FLAGS.OPEN_EXISTING, FILE_FLAGS_AND_ATTRIBUTES.FILE_ATTRIBUTE_NORMAL, default);
+                    handle = PInvoke.CreateFile(pPath, FILE_ACCESS_FLAGS.FILE_GENERIC_READ, FILE_SHARE_MODE.FILE_SHARE_READ, null, FILE_CREATION_DISPOSITION.OPEN_EXISTING, FILE_FLAGS_AND_ATTRIBUTES.FILE_ATTRIBUTE_NORMAL, default);
                 }
 
                 if (!handle.Equals(Constants.INVALID_HANDLE_VALUE))

--- a/src/NerdBank.GitVersioning/ManagedGit/FileHelpers.cs
+++ b/src/NerdBank.GitVersioning/ManagedGit/FileHelpers.cs
@@ -5,8 +5,7 @@ using System.Diagnostics.CodeAnalysis;
 using System.IO;
 using System.Runtime.InteropServices;
 using Microsoft.Win32.SafeHandles;
-using static PInvoke.Kernel32;
-using FileShare = PInvoke.Kernel32.FileShare;
+using Microsoft.Windows.Sdk;
 
 namespace Nerdbank.GitVersioning.ManagedGit
 {
@@ -24,7 +23,7 @@ namespace Nerdbank.GitVersioning.ManagedGit
         {
             if (IsWindows)
             {
-                var handle = CreateFile(path, ACCESS_MASK.GenericRight.GENERIC_READ, FileShare.FILE_SHARE_READ, (SECURITY_ATTRIBUTES?)null, CreationDisposition.OPEN_EXISTING, CreateFileFlags.FILE_ATTRIBUTE_NORMAL, SafeObjectHandle.Null);
+                var handle = PInvoke.CreateFile(path, FILE_ACCESS_FLAGS.FILE_GENERIC_READ, FILE_SHARE_FLAGS.FILE_SHARE_READ, lpSecurityAttributes: null, FILE_CREATE_FLAGS.OPEN_EXISTING, FILE_FLAGS_AND_ATTRIBUTES.FILE_ATTRIBUTE_NORMAL, null);
 
                 if (!handle.IsInvalid)
                 {
@@ -62,12 +61,15 @@ namespace Nerdbank.GitVersioning.ManagedGit
         {
             if (IsWindows)
             {
-                var handle = CreateFile(path, ACCESS_MASK.GenericRight.GENERIC_READ, FileShare.FILE_SHARE_READ, null, CreationDisposition.OPEN_EXISTING, CreateFileFlags.FILE_ATTRIBUTE_NORMAL, SafeObjectHandle.Null);
-
-                if (!handle.IsInvalid)
+                HANDLE handle;
+                fixed (char* pPath = &path[0])
                 {
-                    var fileHandle = new SafeFileHandle(handle.DangerousGetHandle(), ownsHandle: true);
-                    handle.SetHandleAsInvalid();
+                    handle = PInvoke.CreateFile(pPath, FILE_ACCESS_FLAGS.FILE_GENERIC_READ, FILE_SHARE_FLAGS.FILE_SHARE_READ, null, FILE_CREATE_FLAGS.OPEN_EXISTING, FILE_FLAGS_AND_ATTRIBUTES.FILE_ATTRIBUTE_NORMAL, default);
+                }
+
+                if (!handle.Equals(Constants.INVALID_HANDLE_VALUE))
+                {
+                    var fileHandle = new SafeFileHandle(handle, ownsHandle: true);
                     stream = new FileStream(fileHandle, System.IO.FileAccess.Read);
                     return true;
                 }

--- a/src/NerdBank.GitVersioning/NativeMethods.json
+++ b/src/NerdBank.GitVersioning/NativeMethods.json
@@ -1,0 +1,3 @@
+﻿{
+  "$schema": "https://aka.ms/CsWin32.schema.json"
+}

--- a/src/NerdBank.GitVersioning/NativeMethods.txt
+++ b/src/NerdBank.GitVersioning/NativeMethods.txt
@@ -1,0 +1,2 @@
+﻿CreateFile
+INVALID_HANDLE_VALUE

--- a/src/NerdBank.GitVersioning/NerdBank.GitVersioning.csproj
+++ b/src/NerdBank.GitVersioning/NerdBank.GitVersioning.csproj
@@ -12,7 +12,7 @@
     <PackageReference Include="DotNetMDDocs" Version="0.111.0" PrivateAssets="all" Condition=" '$(GenerateMarkdownApiDocs)' == 'true' " />
     <PackageReference Include="LibGit2Sharp" Version="0.27.0-preview-0096" PrivateAssets="none" />
     <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="2.1.0" />
-    <PackageReference Include="Microsoft.Windows.CsWin32" Version="0.1.422-beta" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.Windows.CsWin32" Version="0.1.478-beta" PrivateAssets="all" />
     <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
     <PackageReference Include="Nullable" Version="1.3.0" PrivateAssets="all" />
     <PackageReference Include="Validation" Version="2.5.5-beta" />

--- a/src/NerdBank.GitVersioning/NerdBank.GitVersioning.csproj
+++ b/src/NerdBank.GitVersioning/NerdBank.GitVersioning.csproj
@@ -12,12 +12,12 @@
     <PackageReference Include="DotNetMDDocs" Version="0.111.0" PrivateAssets="all" Condition=" '$(GenerateMarkdownApiDocs)' == 'true' " />
     <PackageReference Include="LibGit2Sharp" Version="0.27.0-preview-0096" PrivateAssets="none" />
     <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="2.1.0" />
+    <PackageReference Include="Microsoft.Windows.CsWin32" Version="0.1.422-beta" PrivateAssets="all" />
     <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
     <PackageReference Include="Nullable" Version="1.3.0" PrivateAssets="all" />
     <PackageReference Include="Validation" Version="2.5.5-beta" />
     <PackageReference Include="Nerdbank.GitVersioning.LKG" Version="3.4.173-alpha" />
     <PackageReference Include="System.Text.Json" Version="4.7.2" />
-    <PackageReference Include="PInvoke.Kernel32" Version="0.7.104" />
   </ItemGroup>
   <ItemGroup>
     <Compile Include="..\Shared\**\*.cs" LinkBase="Shared" />

--- a/src/Nerdbank.GitVersioning.Tasks/Nerdbank.GitVersioning.nuspec
+++ b/src/Nerdbank.GitVersioning.Tasks/Nerdbank.GitVersioning.nuspec
@@ -23,7 +23,6 @@ IMPORTANT: The 3.x release may produce a different version height than prior maj
     <file src="$BaseOutputPath$net461\NerdBank.GitVersioning.dll" target="build\MSBuildFull\NerdBank.GitVersioning.dll" />
     <file src="$BaseOutputPath$net461\Nerdbank.GitVersioning.Tasks.dll" target="build\MSBuildFull\Nerdbank.GitVersioning.Tasks.dll" />
     <file src="$BaseOutputPath$net461\Newtonsoft.Json.dll" target="build\MSBuildFull\Newtonsoft.Json.dll" />
-    <file src="$BaseOutputPath$net461\PInvoke.Kernel32.dll" target="build\MSBuildFull\PInvoke.Kernel32.dll" />
     <file src="$BaseOutputPath$net461\System.Buffers.dll" target="build\MSBuildFull\System.Buffers.dll" />
     <file src="$BaseOutputPath$net461\System.Memory.dll" target="build\MSBuildFull\System.Memory.dll" />
     <file src="$BaseOutputPath$net461\System.Numerics.Vectors.dll" target="build\MSBuildFull\System.Numerics.Vectors.dll" />
@@ -49,7 +48,6 @@ IMPORTANT: The 3.x release may produce a different version height than prior maj
     <file src="$BaseOutputPath$netcoreapp2.1\System.Text.Json.dll" target="build\MSBuildCore\System.Text.Json.dll" />
     <file src="$BaseOutputPath$netcoreapp2.1\Validation.dll" target="build\MSBuildCore\Validation.dll" />
     <file src="$BaseOutputPath$netcoreapp2.1\System.Runtime.CompilerServices.Unsafe.dll" target="build\MSBuildCore\System.Runtime.CompilerServices.Unsafe.dll" />
-    <file src="$BaseOutputPath$netcoreapp2.1\PInvoke.Kernel32.dll" target="build\MSBuildCore\PInvoke.Kernel32.dll" />
 
     <file src="build\Nerdbank.GitVersioning.targets" target="build\Nerdbank.GitVersioning$LKGSuffix$.targets" />
     <file src="build\Nerdbank.GitVersioning.Common.targets" target="build\Nerdbank.GitVersioning.Common.targets" />

--- a/src/Nerdbank.GitVersioning.sln
+++ b/src/Nerdbank.GitVersioning.sln
@@ -1,12 +1,13 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.28404.58
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31411.2
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{4BD1A7CD-6F52-4F5A-825B-50E4D8C3ECFF}"
 	ProjectSection(SolutionItems) = preProject
 		..\.editorconfig = ..\.editorconfig
 		..\.gitignore = ..\.gitignore
+		..\3rdPartyNotices.txt = ..\3rdPartyNotices.txt
 		..\azure-pipelines.yml = ..\azure-pipelines.yml
 		..\build.ps1 = ..\build.ps1
 		Directory.Build.props = Directory.Build.props

